test/mri/ruby/test_dir.rb

```diff
@@ -44,7 +44,7 @@ def test_seek
   end
 
   def test_nodir
-    assert_raise(Errno::ENOENT) { Dir.open(@nodir) }
+    assert_raise(Errno::ENOTDIR) { Dir.open(@nodir) }
```

Comment:
guess, these hasn't been imported for a while.
could you please confirm the test is updated at MRI's for Ruby 2.3 / 2.4 ?

Reply:
I checked this test all the way up to current MRI trunk and it is still ENOENT.